cookbook/profiler/data_collector.rst

```diff
@@ -131,15 +131,15 @@ following example can help you get started:
         </div>
         {% endset %}
 
-        {# Set the "link" value to false if you do not have a big "panel" 
+        {# Set the "link" value to false if you do not have a big "panel"
            section that you want to direct the user to. #}
-        {% include '@WebProfiler/Profiler/toolbar_item.html.twig' with { 'link': true } %}
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', variables = { 'link': true }) }}
```

Comment:
The `variables` name is completelly optional and very verbose. What about dispalying the short version first and mention that, as usual in Twig functions/filters, you can name the variable if you prefer?

```
{{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { 'link': true }) }}
```